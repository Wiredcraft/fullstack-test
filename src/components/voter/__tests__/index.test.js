import React from 'react';
import renderer from 'react-test-renderer';

import Voter from '../index';

test('Voter renders', () => {
  const component = renderer.create(<Voter />);

  expect(component.toJSON()).toMatchSnapshot();
});

test('Voter renders with props', () => {
  const component = renderer.create(
    <Voter {...{ id: 1, rating: 2, onVote: jest.fn() }} />
  );

  expect(component.toJSON()).toMatchSnapshot();
});
