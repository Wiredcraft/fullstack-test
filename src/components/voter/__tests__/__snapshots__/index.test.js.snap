// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voter renders 1`] = `
<div
  className="sc-bdvvtL eLvoKT"
>
  <div
    className="sc-dkPtRN fGaehG"
    onClick={[Function]}
  />
  <div
    className="sc-hKwDye dLhHFB"
    onClick={[Function]}
  />
  <span
    className="sc-gsDKAQ fncrVL"
  />
</div>
`;

exports[`Voter renders with props 1`] = `
<div
  className="sc-bdvvtL eLvoKT"
>
  <div
    className="sc-dkPtRN fGaehG"
    onClick={[Function]}
  />
  <div
    className="sc-hKwDye dLhHFB"
    onClick={[Function]}
  />
  <span
    className="sc-gsDKAQ fncrVL"
  >
    2
  </span>
</div>
`;
